src/utils/update/managing.officer.mapper.ts

```diff
@@ -13,6 +13,7 @@ export const mapToManagingOfficer = (officer: CompanyOfficer): ManagingOfficerIn
 
   return {
     id: raw.links?.self,
+    ch_reference: mapSelfLink(officer.links?.officer?.appointments),
```

Comment:
I think it should be the self link.

Reply:
For now we can use 
const raw = officer.links as any;
raw?.self;
But I should get a PR for a change to the node api sdk up